### DISCUSSION TITLE
Message ID pointing with return path domain URL

### DISCUSSION THREAD

**Discussion by Mathi-S** [postalserver/postal#1916] [Help with using Postal]
we created the mail server when initializing the postal and created a return path URL like rp.postal.example.com

after that, we create a new organization through UI and added a domain as mail.example.com. 
If we are sending mail to users using that domain(mail.example.com), message-id shows as "some_random_string@rp.postal.example.com".
But what we need is message-id should be "some_random_string@mail.example.com".
what kind of setup do we need to do to achieve this. Because we are going to create multiple domains and message-id should different for each domain.

Thanks in advance.

**Comment by willpower232** [comment]
the message ID will already be random so it should be fine, I think there was another issue somewhere else about discussing changing the return path domain but I think it needs to be rp.postal so that it gets delivered to the Postal server and not to your own email system which won't know what to do with it
